### ISSUE TITLE
Rework submit bigram page 

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/sb#27]
In Genesis-SB the Submit Bigram page ( submit-bigram ) uses `[bw_login]` and `[bw_new popst_type="bigram"]` to display a front end form to a logged in user to create a new bigram. 

This needs fixing and reworking for SB

1. The form displays `Undefined: I`  and a plan text entry field instead of `File:` with a "Choose File" button inside and input field of` `type="file"`.
2. The form should be wider; no featured image required.

**Comment by bobbingwide** [comment]
The form didn't work because the oik-media plugin wasn't activated.

**Comment by bobbingwide** [comment]
For the mean time I'm using the incomplete `wide-featured-image` template as a custom template.
Alternatively I could write a `page-submit-bigram.html` template.

**Comment by bobbingwide** [comment]
> For the mean time I'm using the incomplete wide-featured-image template as a custom template.

> Alternatively I could write a page-submit-bigram.html template.

I wrote the `page-submit-bigram.html` template. This doesn't display the sidebar.

I also added a custom template called `singular-one-column.html`

- The one-column refers to the main content. 
- It uses the standard sidebar.
- This custom template is used for pages that use the `[bw_table]` shortcode.
- More often than not these pages don't have a featured image, but the template supports it.
